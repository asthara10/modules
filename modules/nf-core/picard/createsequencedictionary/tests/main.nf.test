nextflow_process {

    name "Test Process PICARD_CREATESEQUENCEDICTIONARY"
    script "../main.nf"
    process "PICARD_CREATESEQUENCEDICTIONARY"

    tag "modules"
    tag "modules_nfcore"
    tag "picard"
    tag "picard/createsequencedictionary"

    test("test_picard_createsequencedictionary") {


        when {
            process {
                """
                
                input[0] = [ 
        [ id:'test' ], // meta map
        file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true) 
    ]

                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out.versions).match("versions") }
                { assert snapshot(process.out).match() }
            )
        }

    }

}
