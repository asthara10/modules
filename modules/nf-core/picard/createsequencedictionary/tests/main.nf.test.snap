{
    "versions": {
        "content": [
            [
                "versions.yml:md5,5c29865a83f407f7e5b57bf64b7acdaf"
            ]
        ],
        "timestamp": "2023-11-24T13:11:20.291243"
    }
}